.github/workflows/_cleanup.yml

```diff
@@ -0,0 +1,62 @@
+name: Cleanup
+
+on:
+  workflow_call:
+    inputs:
+      cluster-name:
+        required: true
+        description: AKS Cluster Name With Randomly Generated Suffix
```

Comment:
You don't need to denote that the cluster name has a randomly generated suffix. Just saying "AKS Cluster Name" is enough.